lib/bootloader_setup.pm

```diff
@@ -334,7 +334,7 @@ sub uefi_bootmenu_params {
       :         send_key 'e';
     # Kiwi in TW uses grub2-mkconfig instead of the custom kiwi config
     # Locate gfxpayload parameter and update it
-    if (is_jeos && (is_tumbleweed || is_sle('>=15-sp2') || is_leap('>=15.2'))) {
+    if (is_jeos && (is_tumbleweed || is_sle('>=15-sp1') || is_leap('>=15.2'))) {
```

Comment:
isn't it would be the same for Leap 15.1 ?

Reply:
Maybe that is an issue on leap side.
@Vogtinator What is your opinion over here, please?

Reply:
Currently Leap 15.1 seems to be using the kiwi generated grub.cfg: https://openqa.opensuse.org/tests/1221767

The maintenance update is pending: https://build.opensuse.org/project/show/openSUSE:Maintenance:12238

I don't think it'll be released today, so it'll probably stay with the old way at least until next week. So if this change is needed for SLE now, I'd just do the change for Leap in a separate PR once it's done.